mod/metrics/lib/javascript/script_metrics.js.coffee

```diff
@@ -65,22 +72,25 @@ $(window).ready ->
 validateWikiRating = (table) ->
   hash = wikiRatingEditorHash table
   valid = tallyWeights table, hash
+  #toEqualize table 
   updateWikiRatingSubmitButton table.closest('form.card-form'), valid
 
 
 DIGITS_AFTER_DECIMAL = 2
 
 tallyWeights = (tbody, hash) ->
   multiplier = 10**DIGITS_AFTER_DECIMAL
-  total =  0
+  total = 0
   valid = true
   $.each hash, (_key, val) ->
-    num = parseFloat val
+    num = parseFloat val 
     total += num * multiplier
-    valid = false unless num > 0
-  total = parseInt(total) / multiplier
+    valid = false if num < 0 || !maxDigit(val)
```

Comment:
we should validate the inputs before we validate the tally.  If the inputs aren't valid, we don't need to do this.